scripts/install.py

```diff
@@ -252,163 +258,112 @@ def action(sim_dir, event_name, run_name, run_dir, vel_mod_dir, srf_dir, srf_sto
-
-        # check if stat_file is empty. empty means user skipped make_obs.sh and is for future events.
-        # use non_uniform_lastest.ll files for future events
-        # stat_file_path is retrived by config, will always be type(string)
-        if stat_file_path == "":
-            # stat_path seems to empty, assigning all related value to latest_ll
-            print "stat_file_path is not specified."
-            print "Using %s" % latest_ll
-            run_stat_dir = os.path.join(stat_dir, event_name)
-            stat_file_path = os.path.join(run_stat_dir, event_name + '.ll')
-            vs30_file_path = os.path.join(run_stat_dir, event_name + '.vs30')
-            vs30ref_file_path = os.path.join(run_stat_dir, event_name + '.vs30ref')
-            # creating sub-folder for run_name
-            # check if folder already exist
-            if not os.path.isdir(run_stat_dir):
-                # folder not exist, creating
-                os.mkdir(run_stat_dir)
-                # making symbolic link to latest_ll
-                cmd = "ln -s %s %s" % (os.path.join(latest_ll_dir, latest_ll + '.ll'), stat_file_path)
-                exe(cmd)
-                # making symbolic link to lastest_ll.vs30 and .vs30ref
-                cmd = "ln -s %s %s" % (os.path.join(latest_ll_dir, latest_ll + '.vs30'), vs30_file_path)
-                exe(cmd)
-                cmd = "ln -s %s %s" % (os.path.join(latest_ll_dir, latest_ll + '.vs30ref'), vs30ref_file_path)
-                exe(cmd)
-        f.write("stat_vs_est = '%s'\n" % vs30_file_path)
-        f.write("stat_vs_ref= '%s'\n" % vs30ref_file_path)
-
-        if stat_file_path is not None:
-            f.write("stat_file='%s'\n" % stat_file_path)
-            f.write("STAT_FILES=[stat_file]\n")
-
-    print dir_list[0]
-    #set_permission(dir_list[
-    #                   0])  # if user_root is first time created, recursively set permission from there. otherwise, set permission from sim_dir
-
-    sys.path.append(sim_dir)
-
-    show_horizontal_line(c='*')
-
-    # create model_params
-    if yes_model_params:
-        print "Producing model params. It may take a minute or two"
-        from gen_coords import gen_coords
-        gen_coords()
-        print "Done"
-        # else:
-        # print "Generation of model params is skipped. You need to fix params_base.py manually"
-
-    # currently not generating statgrid on the fly, using pregenerated .ll
-    # if stat_file_path is None:
-    #    show_horizontal_line(c='*')
-    #
-    #    print "Producing statgrid. It may take a minute or two"
-    #
-    #    import gen_statgrid
-    #    stat_file_path = gen_statgrid.main(outDir=sim_dir)
-    #    print "Done"
-    #    with open(os.path.join(sim_dir,"params_base.py"),"a") as f:
-    #        f.write("stat_file='%s'\n" %stat_file_path)
-    #        f.write("STAT_FILES=[stat_file]\n")
-    #
-    #    print "Statgrid: %s" %stat_file_path
-
-
-
-    show_horizontal_line(c='*')
```

Comment:
i think it is better to keep this line, so the user can tell what is going on more easily